src/Log/Doctrine/EntityUpdateEvent.php

```diff
@@ -73,6 +73,7 @@ public function getNewChanged(): array
 
     public function getTitle(): string
     {
+        assert(is_string($this->getEntityType()));
```

Comment:
The type hint for `getEntityType()` specifies that it's value may be null. Therefore, this assert doesn't necessarily hold true.

Reply:
This issue occurs a couple times throughout the codebase as well, please review the changes with this in mind.

Reply:
while `getEntityType` might return a null, `fqcnToName` doesn't accept null as an argument. So for a successful call we do need it to be a string. correct?

Reply:
That's correct. However, as the value may not be a string, either an update of the respective type definitions or an explicit check would be more appropriate, eg.
```php
if (!is_string($this->getEntityType())) throw new \Exception();
```